autoprofile.py

```diff
@@ -33,32 +42,105 @@ def __init__(self):
         self.name = _("Automatic Profile")
         self.bio_enabled = False
         self.name_enabled = False
+        self.pfp_enabled = False
+        self.pfp_remove_latest = None
         self.raw_bio = None
         self.raw_name = None
+        self.pfp_degree = 0
 
     async def client_ready(self, client, db):
         self.client = client
 
+    async def autopfpcmd(self, message):
+        """Rotates your profile picture every n seconds with x degrees, usage:
+           .autopfp <timeout> <degrees> <remove previous (last pfp)>
+
+           Timeout - seconds
+           Degrees - 60, -10, etc
+           Remove last pfp - True/False, case sensitive"""
+
+        if not pil_installed:
+            return await utils.answer(message, _("<b>You don't have PIL (Pillow) installed.</b>"))
+
+        if not await self.client.get_profile_photos('me', limit=1):
+            return await utils.answer(message, _("<b>You don't have a profile pic set.</b>"))
+
+        msg = utils.get_args(message)
+        if len(msg) != 3:
+            return await utils.answer(message, _("<b>Autopfp requires three args. See the help for syntax.</b>"))
+
+        try:
+            timeout_autopfp = int(msg[0])
+        except ValueError:
+            return await utils.answer(message, _("<b>Wrong time.</b>"))
+
+        try:
+            degrees = int(msg[1])
+        except ValueError:
+            return await utils.answer(message, _("<b>Wrong degrees value.</b>"))
+
+        try:
+            delete_previous = literal_eval(msg[2])
+        except (ValueError, SyntaxError):
+            return await utils.answer(message, _("<b>Please pass True or False for previous pfp removal.</b>"))
+
+        pfp = BytesIO()
+        await self.client.download_profile_photo('me', file=pfp)
+        raw_pfp = Image.open(pfp)
+
```

Comment:
now run pfp.close() to release resources

Reply:
nope won't work, errors `ValueError: I/O operation on closed file.`

Reply:
Then only close it after all io ops are done